cpp/src/arrow/compute/api_scalar.h

```diff
@@ -882,6 +891,20 @@ ARROW_EXPORT
 Result<Datum> Round(const Datum& arg, RoundOptions options = RoundOptions::Defaults(),
                     ExecContext* ctx = NULLPTR);
 
+/// \brief Round a value to a given precision.
+///
+/// If argument is null the result will be null.
+///
+/// \param[in] arg1 the value rounded
+/// \param[in] arg2 the number of significant digits to round to
+/// \param[in] options rounding options (rounding mode and number of digits), optional
```

Comment:
```suggestion
/// \param[in] options rounding options (rounding mode), optional
```
Or just get rid of the parentheses section entirely.

Reply:
Done.